client/js/components/fragments/MiniProfile.js

```diff
@@ -1,15 +1,35 @@
 import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 
+import DefaultAvatar from '../../../img/avatar.jpg';
+import 'file?name=[name].[ext]!../../../img/avatar.jpg';
+
 class MiniProfile extends Component {
   render() {
     const { user } = this.props;
 
+    let fullName;
+    if (user.firstName && user.lastName) {
```

Comment:
Could you move that logic to separate function? So that render() is as simple as possible